OrbitService/Process.cpp

```diff
@@ -19,7 +19,22 @@
 
 namespace orbit_service {
 
-ErrorMessageOr<Process> Process::FromPidAndCpuUsage(pid_t pid, double cpu_usage) {
+void Process::UpdateCpuUsage(utils::Jiffies process_cpu_time, utils::Jiffies total_cpu_time) {
+  const auto diff_process_cpu_time =
+      static_cast<double>(process_cpu_time.value - previous_process_cpu_time_.value);
+  const auto diff_total_cpu_time =
+      static_cast<double>(total_cpu_time.value - previous_total_cpu_time_.value);
+
+  const auto cpu_usage = std::clamp(diff_process_cpu_time / diff_total_cpu_time, 0.0, 1.0);
+
+  // TODO(hebecker): Rename cpu_usage to cpu_usage_percent. Not being normalized was surprising
```

Comment:
interesting - maybe we should do the opposite and use rate here - normalize on the client-side.

Reply:
Good point. I adjusted the comment.